src/GeohashHilbert.jl

```diff
@@ -299,7 +331,13 @@ function neighbours(geohash, bits_per_char)
 
 end
 
-function rectangle(geohash, bits_per_char)
+"""
+    rectangle(geohash, bits_per_char = 2)
+
+Return a GeoJSON Dict which encodes as a Feature the rectangle associated with a given
```

Comment:
that* (instead of which), as there is not leading comma

Reply:
Is this a rule? Which only after a preceding comma? I did not know this.